### PR TITLE
FIR checker: ABSTRACT_PROPERTY_IN_PRIMARY_CONSTRUCTOR_PARAMETERS

### PR DESCRIPTION
**Pull request by tgeng**

### CODE REVIEW

File: compiler/testData/diagnostics/tests/properties/abstarctPropertyInPrimaryConstructor.fir.kt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class A(<!ABSTRACT_PROPERTY_IN_PRIMARY_CONSTRUCTOR_PARAMETERS!>abstract<!> val i: Int)
+class B(abstract i: Int)
```
Comment (jsjeon) [inline compiler/testData/diagnostics/tests/properties/abstarctPropertyInPrimaryConstructor.fir.kt:2]: Missing `WRONG_MODIFIER_TARGET`? cc @punzki

**Review by jsjeon — APPROVED** [review]

**Comment by dzharkov** [comment]
Please, rebase your changes on fresh master

**Comment by tgeng** [comment]
Thanks! Rebased.

**Comment by dzharkov** [comment]
Thank you for your contribution! Merged manually

**Pull request closed without merging**